security/pfSense-pkg-openvpn-client-export/files/usr/local/pkg/openvpn-client-export.inc

```diff
@@ -944,7 +944,7 @@ function openvpn_client_export_sharedkey_config($srvid, $useaddr, $proxy, $nokey
 	}
 }
 
-function openvpn_client_export_build_remote_lines($settings, $useaddr, $interface, $expformat, $nl) {
+function openvpn_client_export_build_remote_lines($settings, $useaddr, $interface, $expformat, $nl, $legacy) {
```

Comment:
Would be best to make `$legacy` default to `false` since it's being added to an existing function.

Reply:
ok, fixed